src/test/java/org/opentripplanner/routing/algorithm/TestFares.java

```diff
@@ -67,6 +63,7 @@ public void testPortland() throws Exception {
         options.dateTime = startTime;
         options.setRoutingContext(gg, feedId + ":10579", feedId + ":8371");
         // from zone 3 to zone 2
+        options.maxTransfers = 3;
```

Comment:
This is now set everywhere ot different values and it is hard to keep track of current "default" value. Please resolve it instead of adding one more value.

Reply:
I will resolve it, however this one is test specific.